### PR TITLE
Add automatic tagging

### PR DESCRIPTION
**Pull request by joebochill**
#### Changes proposed in this Pull Request:
- Add automatic tagging command to circleCI

Putting this in before merging the patch version to see if it works.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -78,6 +79,18 @@ jobs:
...
             cd login-workflow && yarn publish:package -b $CIRCLE_BRANCH
 
+  gh_tag_login_workflow:
+    docker:
+      - image: circleci/node:12.16.3-browsers
+    steps:
+      - checkout
+      - gh/setup
+      - run:
+          name: Tag @pxblue/publish
```
Comment (emclaug2) [inline .circleci/config.yml:89]: Rename this to "Tag Workflow"

File: .circleci/config.yml [hunk 2]
```diff
@@ -78,6 +79,18 @@ jobs:
...
+          command: |
+            cd pxb-publish
```
Comment (emclaug2) [inline .circleci/config.yml:91]: cd into the root folder here

Comment (joebochill) [reply .circleci/config.yml:91]: this is why I assigned you

**Comment by jeffvg** [comment]
should we be using github-cli@1.0.3 and not 1.0?

**Comment by emclaug2** [comment]
> should we be using github-cli@1.0.3 and not 1.0?

1.0 seems to be working but I don't think it could hurt to use the latest.

**Review by emclaug2 — APPROVED** [review]

**Comment by joebochill** [comment]
I'm not sure how the orb versioning works..."1.0" may install "1.0.3" already

**Pull request merged**